src/security/cli_security.c

```diff
@@ -144,6 +144,7 @@ process_credential_response(Drpc__Response *response,
 
 	if (response->status != DRPC__STATUS__SUCCESS) {
 		/* Recipient could not parse our message */
+		D_ERROR("Response status is: %d\n", response->status);
```

Comment:
Since it is a D_ERROR, it should really be more explicit.

Reply:
Can you clarify what you mean by more explict?